source/PlayerInfo.cpp

```diff
@@ -4282,6 +4308,18 @@ void PlayerInfo::Save(DataWriter &out) const
 	// Save any "primary condition" flags that are set.
 	conditions.Save(out);
 
+	// Save raid fleets that got stacked up.
```

Comment:
```suggestion
	// Save raid fleets as they get stacked up.
```

This may be a bit clearer here?